### ISSUE TITLE
Failling while generating the VM

### ISSUE THREAD

**Issue by felixveysseyre** [Kitware/HPCCloud-deploy#105]
Hello,

I am trying to generate the main VM.

In order to do that, I have executed the following commands:
`vagrant up`

It run for a few minutes and I got the following error:
```
TASK [girder : Build Girder (web)] *********************************************
task path: /Users/felixveysseyre/Projects/XXXX/HPCCD/ansible/roles/girder/tasks/main.yml:63
fatal: [hpccloud-vm]: FAILED! => {"changed": true, "cmd": ["/usr/local/bin/girder-install", "web", "--all-plugins"], "delta": "0:00:00.932344", "end": "2017-10-11 11:31:17.123491", "failed": true, "rc": 1, "start": "2017-10-11 11:31:16.191147", "stderr": "ERROR: npm is known not to run on Node.js v4.4.5\nNode.js 4 is supported but the specific version you're running has\na bug known to break npm. Please update to at least ${rel.min} to use this\nversion of npm. You can find the latest release of Node.js at https://nodejs.org/\nTraceback (most recent call last):\n  File \"/usr/local/bin/girder-install\", line 11, in <module>\n    load_entry_point('girder', 'console_scripts', 'girder-install')()\n  File \"/opt/hpccloud/girder/girder/utility/install.py\", line 362, in main\n    parsed.func(parsed)\n  File \"/opt/hpccloud/girder/girder/utility/install.py\", line 214, in install_web\n    plugins=opts.plugins, noPlugins=opts.no_plugins)\n  File \"/opt/hpccloud/girder/girder/utility/install.py\", line 181, in runWebBuild\n    (' '.join(cmd), proc.returncode))\nException: Web client install failed: `npm install --unsafe-perm --no-save --production` returned 1.", "stderr_lines": ["ERROR: npm is known not to run on Node.js v4.4.5", "Node.js 4 is supported but the specific version you're running has", "a bug known to break npm. Please update to at least ${rel.min} to use this", "version of npm. You can find the latest release of Node.js at https://nodejs.org/", "Traceback (most recent call last):", "  File \"/usr/local/bin/girder-install\", line 11, in <module>", "    load_entry_point('girder', 'console_scripts', 'girder-install')()", "  File \"/opt/hpccloud/girder/girder/utility/install.py\", line 362, in main", "    parsed.func(parsed)", "  File \"/opt/hpccloud/girder/girder/utility/install.py\", line 214, in install_web", "    plugins=opts.plugins, noPlugins=opts.no_plugins)", "  File \"/opt/hpccloud/girder/girder/utility/install.py\", line 181, in runWebBuild", "    (' '.join(cmd), proc.returncode))", "Exception: Web client install failed: `npm install --unsafe-perm --no-save --production` returned 1."], "stdout": "", "stdout_lines": []}
	to retry, use: --limit @/Users/felixveysseyre/Projects/XXXX/HPCCD/ansible/site.retry

PLAY RECAP *********************************************************************
hpccloud-vm                : ok=90   changed=61   unreachable=0    failed=1   

Ansible failed to complete successfully. Any error output should be
visible above. Please fix these errors and try again.
```
I am using:

The latest version of _HPCCloud-deploy_: 3c02255a7958809a006f79934d803854a551dc10
MacOS: 10.11.2 (15C50)
VirtualBox: 5.1.22 r115126 (Qt5.6.2)

Thanks for you help !

**Comment by cjh1** [comment]
Looks like Girder now need node > 6, this should fix it #106

**Comment by felixveysseyre** [comment]
Thanks @cjh1, it fixed the issue !
